### PR TITLE
Adding additional sanity checks on divide by zeros in voice-quality-s…

### PR DESCRIPTION
**Pull request by shkyang**
…tats, started removing member variables in voice-quality-extractor.

### CODE REVIEW

File: src/feat/feature-voice-quality-statistics.cc [hunk 1]
```diff
@@ -12,21 +12,30 @@ VoiceQualityStatisticsComputer::~VoiceQualityStatisticsComputer()
 
 BaseFloat VoiceQualityStatisticsComputer::calcAvg(const vector<BaseFloat> &params)
 {
+    if (params.size() <= 1)
+    {
+        throw std::runtime_error("Error in calcAvg, input vector needs to be size 1 or above for our calculation to work.");
+    }
     BaseFloat sum = std::accumulate(params.begin(), params.end(), 0.0);
-    return sum / static_cast<BaseFloat>(params.size());
+    return sum / static_cast<BaseFloat>(params.size() - 1);
```
Comment (jangwonkim) [inline src/feat/feature-voice-quality-statistics.cc]: why average has -1?

File: src/feat/feature-voice-quality-statistics.cc [hunk 2]
```diff
@@ -12,21 +12,30 @@ VoiceQualityStatisticsComputer::~VoiceQualityStatisticsComputer()
...
-        skewness += (*i - avg) * (*i - avg) * (*i - avg);
-        kurt += (*i - avg) * (*i - avg) * (*i - avg) * (*i - avg);
+        const BaseFloat diff = (*i-avg);
+        var += diff * diff;
+        skewness += diff * diff * diff;
+        kurt += diff * diff * diff * diff;
     }
 
-    var = var / (params.size() - 1);
+    var = var / (params.size());
```
Comment (jangwonkim) [inline src/feat/feature-voice-quality-statistics.cc]: why var doesn't have -1?

File: src/feat/feature-voice-quality-statistics.cc (hunk 1 above)
Comment (shkyang) [inline src/feat/feature-voice-quality-statistics.cc]: oh that's my mistake, meant to put back in the, I'll fix it.

File: src/featbin/compute-kaldi-voice-qual-stat-params.cc [hunk 3]
```diff
@@ -1,355 +0,0 @@
-// Copyright 2016 Cobalt Speech and Language (authors: Mohammad Ariful Haque)
```
Comment (fahim-zubayer) [inline src/featbin/compute-kaldi-voice-qual-stat-params.cc]: The makefile for this directory needs to be changed to exclude this file. Caused me an error.

**Pull request merged**